backend/python/app/graphql/mutations/story_mutation.py

```diff
@@ -78,6 +81,7 @@ class Arguments:
 
     story = graphene.Field(lambda: CreateStoryTranslationResponseDTO)
 
+    @require_authorization_by_role_gql({"Admin"})
```

Comment:
should require both roles - user calls this from the homepage when signing up to review a story translation